firefox/firefox-git.dockerfile

```diff
@@ -4,10 +4,11 @@ MAINTAINER Jan Keromnes "janx@linux.com"
 # Install Firefox build dependencies.
 # One-line setup command from:
 # https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Linux_Prerequisites#Most_Distros_-_One_Line_Bootstrap_Command
-RUN sudo apt-get update -q \
+RUN sudo apt-get update \
```

Comment:
It's not needed and in the Docker library images it's never used. Often when an error occurs it would be more useful if we had the context (which is suppressed with -q).